std/zig/tokenizer.zig

```diff
@@ -673,6 +687,9 @@ pub const Tokenizer = struct {
                         seen_escape_digits = 0;
                         expected_escape_digits = 6;
                     },
+                    'n', 'r', '\\', 't', '\'', '\"' => {
+                        // TODO
```

Comment:
Can you make this `@panic("TODO");`? Better to crash here than to silently be incorrect.